conanfile.py

```diff
@@ -151,6 +152,10 @@ def build_requirements(self):
             if not tools.which('pkg-config'):
                 self.build_requires('pkg-config_installer/0.29.2@bincrafters/stable')
 
+    def config_options(self):
+        if self.settings.os == "Windows":
```

Comment:
We should maybe go even further`if self.settings.os != "Linux":` , because it's the only working configuration known currently.